app/src/common/css/variables/colors.scss

```diff
@@ -16,12 +16,12 @@
 
 /* stylelint-disable color-no-hex */
 
-$COLOR--system-issue: #0274d1;
+$COLOR--system-issue: #3e7be6;
 $COLOR--passed: #56b985;
-$COLOR--product-bug: #ec3900;
-$COLOR--automation-bug: #f7d63e;
-$COLOR--to-investigate: #ffb743;
-$COLOR--no-defect: #aaaaaa;
+$COLOR--product-bug: #d32f2f;
+$COLOR--automation-bug: #ffc208;
+$COLOR--to-investigate: #00829b;
+$COLOR--no-defect: #cfcfcf;
```

Comment:
Please use colors from `new-colors.scss`. If necessary color doesn't exists, define it.